packages/ui/src/Customization/CustomizationsExtensionDetail.css

```diff
@@ -0,0 +1,16 @@
+.extension-group:not(:last-child) {
+  margin-bottom: 35px;
+}
+
+.extension-id {
+  font-size: 0.5em;
+  margin-left: 10px;
+}
+
+.extension-title {
+  margin: 0;
+}
+
+button + button {
```

Comment:
Can you refactor these classes to use BEM, and namespace that button + button thing?